SCClassLibrary/Common/Streams/Rest.sc

```diff
@@ -1,56 +1,23 @@
-Rest {
-	var <>dur = 1;
-	*new { |dur(1)|
-		^super.newCopyArgs(dur)
-	}
-	// for use by Pfunc and Pfuncn
-	*processRest { |inval|
-		inval.put(\isRest, true);
-		^1
-	}
-	processRest { |inval|
-		inval.put(\isRest, true);
-		^dur.value(inval)
-	}
-	// for use e.g. in ListPatterns
-	*embedInStream { |inval|
-		^this.processRest(inval).yield;  // the actual return value is the next inval
-	}
-	embedInStream { |inval|
-		^this.processRest(inval).yield;
-	}
-	*asStream {
-		^Routine({ |inval|
-			loop {
-				inval = this.embedInStream(inval);
-			}
-		})
-	}
-	asStream {
-		^Routine({ |inval|
-			loop {
-				inval = this.embedInStream(inval);
-			}
-		})
+
+Rest : Operand  {
+
+	*new { |value = 1|
+		^super.newCopyArgs(value.dereferenceOperand).unwrapBoolean
```

Comment:
Functionally, `super.newCopyArgs(value.dereferenceOperand)` and `super.new(value.dereferenceOperand)` will be the same here -- `newCopyArgs` begins with inherited instance variables (i.e. it isn't necessary for a class to declare its own unique variables to use `newCopyArgs`).

If it's going to be a new style recommendation to avoid newCopyArgs for inherited variables, we can discuss that as part of the style guide, but maybe not hold up this PR over this.